<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CornerWidget</class>
 <widget class="QWidget" name="CornerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="themeBtn">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="settingBtn">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="donationBtn">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="githubBtn">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="aboutBtn">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
